src/lib/containers/download_list_v2/DownloadCartPageWrapper.tsx

```diff
@@ -2,18 +2,18 @@ import React from 'react'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { useSynapseContext } from '../../utils/SynapseContext'
 import { SynapseErrorBoundary } from '../ErrorBanner'
-import DownloadListStats from './DownloadListStats'
+import DownloadCartPage from './DownloadCartPage'
 
 /**
- * Show the count, and total size, of the files that the user has added to their Download Cart (only includes files are currently available for download).
+ * Show the Download Cart page.
  * "Wrapped" by a query client provider (for caching) and an error boundary.
  */
-export default function DownloadListStatsWrapper() {
+export default function DownloadCartPageWrapper() {
   const { accessToken } = useSynapseContext()
   const queryClient = new QueryClient({
     defaultOptions: {
       queries: {
-        staleTime: 60 * 1000, // 1m
+        staleTime: 20 * 1000, // 1m
```

Comment:
Is `useGetDownloadListStatistics` in the DownloadCartPage the only hook that uses this query client? Do you need that in its own cache? You could specify a shorter stale time as a parameter in the hook function if you need that.

Reply:
I'll just remove the wrapper!